### ISSUE TITLE
`neonOS::removeDups` perplexing error

### ISSUE THREAD

**Issue by znickerson8** [NEONScience/NEON-OS-data-processing#5]
Getting the following error:

```
Error in neonOS::removeDups(data = apc_taxonomyRaw, variables = variables_20072, :
Input data appear to be the basic download package. The expanded data package is required for removeDups() to identify all duplicates correctly.
```

Even though we are downloading the expanded download package

Reproducible code:

```
# Set date variables
startDate_checked = "2021-07"
endDate_checked = "2021-07"
# Load portal data
apc_allTabs <- loadByProduct(
  dpID='DP1.20072.001',
  check.size=F, 
  site = "all",
  package='expanded',
  startdate = startDate_checked,
  enddate = endDate_checked,
  token = Sys.getenv('NEON_PAT')) 
list2env(apc_allTabs, envir=.GlobalEnv)
```

**Comment by znickerson8** [comment]
@stephparker @cklunch

**Comment by cklunch** [comment]
@znickerson8 @stephparker The problem is that phylum and infraspecificEpithet are in the variables file but not in the data. Were they added to the pub workbook recently? The most likely way we'd end up in this situation is if the workbook was updated and the data product was re-published but not re-processed.

**Comment by stephparker** [comment]
@cklunch yes but that issue happened in apc_taxonomyProcessed as well and I didn't get this error, I got the error saying the fields were missing.

**Comment by stephparker** [comment]
@cklunch @znickerson8 ok when I add those columns, this does run. I'm not sure why the error message was different from the other dataframes.

**Comment by cklunch** [comment]
@stephparker Yeah, if the code sees missing fields from the basic package, it says they're missing, but if it sees missing fields from the expanded package, it suggests maybe you're working with the basic package. I'll improve the messaging in the next version, thanks for the heads up!
